src/components/main-page/Hero.tsx

```diff
@@ -0,0 +1,101 @@
+import CellImage from "@/assets/cell.png";
+import Image from "next/image";
+import { AnimatePresence, motion } from "framer-motion";
+import { useRef, useState } from "react";
+import Link from "next/link";
+
+export default function Hero() {
+  return (
+    // Landing hero
+    <div>
+      <div
+        className="flex flex-row flex-wrap items-center justify-center
+                xl:flex-nowrap "
+      >
+        <Image
+          className="xl:w-[50rem] xl:h-[50rem]"
```

Comment:
Use percentage sizing for more responsive image size ex. w-[40%] or w-[40dvw]

Also not sure if I remember correctly, but you might be fine with just setting w or just setting h, but idk